src/features/Apiexplorer/Schema/RecursiveContent/SchemaBodyHeader/index.tsx

```diff
@@ -110,7 +112,7 @@ const SchemaBodyHeader = ({
                 <div className={styles.schemaObjectContent}>
                   <div>
                     <button onClick={() => setIsOpenObject(!is_open_object)}>
-                      {title ? key_value : 'object'}
+                      {title ? key_value : items_type ? items_type : 'object'}
```

Comment:
`{title ? key_value : items_type ??  'object'}`

Reply:
Thank you 👍